src/extractor/guidance/coordinate_extractor.cpp

```diff
@@ -352,40 +370,25 @@ util::Coordinate CoordinateExtractor::ExtractRepresentativeCoordinate(
          * We distinguish between turns that simply model the initial way of getting onto the
          * destination lanes and the ones that performa a larger turn.
          */
-        const double offset =
-            std::min(0.5 * considered_lanes * ASSUMED_LANE_WIDTH, 0.2 * segment_distances.back());
-        coordinates = TrimCoordinatesToLength(std::move(coordinates), offset, segment_distances);
+        coordinates =
+            TrimCoordinatesToLength(std::move(coordinates),
+                                    2 * distance_to_skip_over_due_to_coordinate_inaccuracies,
```

Comment:
:D or name it `distance_epsilon` and put the explanation in a comment.

gotta agree with danpat tho. its intent is very clear.